api/tests/routes/pro/post_collective_offers_test.py

```diff
@@ -17,7 +17,8 @@ def test_create_collective_offer(self, client):
         venue = offerers_factories.VenueFactory()
         template = educational_factories.CollectiveOfferTemplateFactory(venue=venue)
         offerer = venue.managingOfferer
-        offerers_factories.UserOffererFactory(offerer=offerer, user__email="user@example.com")
+        user_offerer = offerers_factories.UserOffererFactory(offerer=offerer, user__email="user@example.com")
+        user = user_offerer.user
```

Comment:
```suggestion
        user = offerers_factories.UserOffererFactory(offerer=offerer, user__email="user@example.com").user
```
Si tu veux être plus concis